backend/coreapp/compilers.py

```diff
@@ -502,6 +508,27 @@ def available_platforms() -> List[Platform]:
     cc=SATURN_CC,
 )
 
+DREAMCAST_CC = (
```

Comment:
can this be tidied up? 

could/should any of those compiler flags be *not* hard-coded?